docs/mine-hnt/miner-cli/authorize.mdx

```diff
@@ -0,0 +1,186 @@
+---
+id: minercli-authorize
+sidebar_label: authorize
+hide_title: true
```

Comment:
Let's make this false or remove it entirely on every doc in this PR, we want the titles to render.